commons/clever-cloud-apis/cc-api.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Clever Cloud API
+position: 1
+---
+# Clever Cloud API
+
+Clever Cloud console is built on top of an API that allow you to manage your
+account and products. This article will explain how to connect to this API and
+use it.
+
+## Make a request on the API
+
+### Authentication
+
+Clever Cloud API works with an OAuth based authentication. First, you need to
+[create some tokens](https://console.clever-cloud.com/users/me/tokens) in
+the Clever Cloud console. Use these tokens to make an OAuth authentication.
+
```

Comment:
Please describe the dance you need to perform to obtain the access token (not in full detail, just the different calls you have to make, add a link to the description of a oauth1 dance)